src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,114 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding: 50px 40px;
+
+  // @include catalog-card;
+
+  &__card {
+    display: flex;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+
+    background: $white-color;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+
+    padding: 32px 16px 16px;
+    width: 200px;
+
+    &-page {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 160px;
+      height: 134px;
+      margin-bottom: 40px;
+    }
+
+    &-title {
+      font-weight: 500;
+      font-size: 12px;
+      line-height: 18px;
+      margin-bottom: 4px;
+      color: $bludark-color;
+    }
+
+    &-product-code {
+      width: $width-section;
+      text-align: left;
+      margin-bottom: 16px;
+      font-size: 10px;
+      line-height: 14px;
+      color: #616070;
+    }
+
+    &-rating {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-bottom: 24px;
+      width: $width-section;
+    }
+
+    &-reviews {
+      font-size: 10px;
+      color: $bludark-color;
+    }
+
+    &-price {
+      display: flex;
+      justify-content: space-between;
+      color: $bludark-color;
+      margin-bottom: 16px;
+      line-height: 18px;
+      width: $width-section;
+
+      &-text {
+        font-size: 12px;
+        color: #616070;
+      }
+
+      &-value {
+        font-weight: $bold-weight;
+        font-size: 16px;
+      }
+    }
+
+    &-link {
+      font-weight: $bold-weight;
+      font-size: 14px;
+      padding: 10px 70px;
+
+      text-decoration: none;
+      text-transform: uppercase;
+      color: $white-color;
+
+      background-color: $arial-color;
+      border: 1px solid $arial-color;
+      border-radius: 5px;
+    }
+
+    &-link:hover {
+      background-color: $white-color;
+      color: $arial-color;
+    }
+
+    .stars {
+      display: flex;
```

Comment:
The 'stars' class should be a separate block following BEM methodology. Consider creating a separate file for it, like '_stars.scss' and import it in the main SCSS file.